### PR TITLE
feat(website): implement home screen, colors, and main content layout

### PR DESCRIPTION
**Pull request by emmenko**
- implements home screen
- adjust main website layout
- define some basic colors palette
- add necessary plugins
- setup netlify

### CODE REVIEW

**Comment by emmenko** [comment]
The site preview is deployed, in case you want to check it out

**Comment by montezume** [comment]
Looks pretttttty nice.  🎉

File: website/src/colors.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/prefer-default-export
+export const light = {
```
Comment (emmenko) [inline website/src/colors.js]: We don’t do theming now but having already a bit of a structure does not hurt.

Also, I’m still playing around with the colors choices and names, so it’s not final yet. I will go through them with the design team as well.

File: website/src/components/code-block.js [hunk 2]
```diff
@@ -1,15 +1,135 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Highlight, { defaultProps } from 'prism-react-renderer';
+import { customProperties } from '@commercetools-frontend/ui-kit';
+import * as colors from '../colors';
+
+// TODO: improve colors
+const theme = {
```
Comment (emmenko) [inline website/src/components/code-block.js]: This is the “theme” config for the code syntax highlighter. It’s still a WIP at the moment, in terms of color choices.

File: website/src/components/markdown-layout.js [hunk 3]
```diff
@@ -2,10 +2,67 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { MDXProvider } from '@mdx-js/react';
 import { css } from '@emotion/core';
+import styled from '@emotion/styled';
 import Layout from './layout';
 import SEO from './seo';
 
-const components = {};
+// TODO: style components
```
Comment (emmenko) [inline website/src/components/markdown-layout.js]: Will work on that in a follow up PR

File: website/src/pages/404.js [hunk 4]
```diff
@@ -1,13 +1,37 @@
...
+        justify-content: center;
+      `}
+    >
+      <h1>Page not found</h1>
+      <div
+        css={css`
+          max-width: 50%;
+        `}
+      >
+        <Lottie
```
Comment (emmenko) [inline website/src/pages/404.js]: This is to render the animated SVG

File: website/src/pages/index.js [hunk 5]
```diff
@@ -1,22 +1,222 @@
...
+
+const heroSlides = [
+  { id: 0, component: DevelopItSvg },
+  { id: 1, component: DeployItSvg },
+  { id: 2, component: RegisterItSvg },
+];
+
+const IndexPage = () => {
+  const [slideIndex, nextSlide] = React.useState(0);
+  const transitions = useTransition(slideIndex, i => i, {
```
Comment (emmenko) [inline website/src/pages/index.js]: react-spring is pretty cool

**Comment by celeste-horgan** [comment]
Beautiful as usual, Nicola!

But: if you're going to use the geometric code-like font as a header, I'd strongly suggest falling back to Open Sans as body text. Particularly given that the App Kit will have lots of codeblocks and `code highlights`, it's going to get unpleasant and difficult to read very quickly if you don't. A large part of the reason we use geometric fonts for code highlighting is to create a visual difference between code and body text, so that big documents are easier to scan.

On another note: Since you've started tackling this website, do we need to start a conversation about writing some of the overview/philosophy type text for this site? We spoke about it briefly a few months ago, but you weren't ready to work on the site so we stopped talking.

**Comment by emmenko** [comment]
@celeste-horgan yes thanks for you feedback. At the moment I'm mostly focused on the basic structure, layout parts and colors. Once I'm done with that we can focus on the content part. There will be for sure other details to adjust and discuss, so I'll be happy to look at those things together with you.

**Pull request merged**